src/latex.jl

```diff
@@ -139,24 +147,89 @@ end
 
 # \todo update LaTeXSVG cache to use output of strip_eq as the key. See https://github.com/JuliaAnimators/Javis.jl/pull/307#issuecomment-749616375
 function get_latex_svg(text::LaTeXString)
-    # check if it's cached
+    #check if it's cached
+    """ for now the default is to use tex2svg, from mathjax
+    pass `Javis.setLaTeXprog(:dvisvgm)` to yse dvisvgm
+    """
+
     if haskey(LaTeXSVG, text)
         svg = LaTeXSVG[text]
     else
-        ts = replace(strip_eq(text), "\n" => " ")
-        command = if Sys.iswindows()
-            `cmd /C tex2svg $ts`
-        else
-            `tex2svg $ts`
-        end
-        try
-            svg = read(command, String)
-        catch e
-            @warn "Using LaTeX needs the program `tex2svg` which might not be installed"
-            @info "It can be installed using `npm install -g mathjax-node-cli`"
-            throw(e)
+        if Javis.LaTeXprog == :tex2svg
+            ts = replace(strip_eq(text), "\n" => " ")
+            command = if Sys.iswindows()
+                `cmd /C tex2svg $ts`
+            else
+                `tex2svg $ts`
+            end
+            try
+                svg = read(command, String)
+            catch e
+                @warn "Using LaTeX needs the program `tex2svg` which might not be installed"
+                @info "It can be installed using `npm install -g mathjax-node-cli`,or if you have a TeX distribution installed try setting `setLaTeXprog(:dvisvgm)`"
+                throw(e)
+            end
+        elseif Javis.LaTeXprog == :dvisvgm
+            println("using dvisvgm")
+            svg = tex2svg(text)
         end
         LaTeXSVG[text] = svg
     end
     return svg
 end
+
+function tex2svg(text::LaTeXString; output_dir = "./.TeXfolder")
+    """
+    generates svg from TeX;
+    default folder placed is ./.TeXfolder,
+    pdf's and aux files are deleted , tex and log files and other files remain
+    add LaTeX packages in `packages=[]`
```

Comment:
I don't understand the comment. Is there a reason why they don't get deleted? 
Also what do you mean with `packages =[]` here?

Reply:
there's no reason to keep the pdf's around after we're done converting to svg;
i just kept it around incase people want to inspect the log and tex files if they do run into errors

Reply:
`packages=[]` was an earlier way i was passing the packages to be added in the usepackage line of the .tex file ; ive changed it in a later commit to a module level variable `Javis.LaTeXusepackages` which is a vector holding all the package names.
will remove the `add Latex package in ...` line